tools/compare.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+import unittest
```

Comment:
Can you please run the autopep on git master and submit it as a pr, so this pr will be formatting-change-only clean?

Reply:
Done. #739